hbase-common/src/main/java/org/apache/hadoop/hbase/util/GsonUtil.java

```diff
@@ -43,7 +43,7 @@ private GsonUtil() {
    */
   public static GsonBuilder createGson() {
     return new GsonBuilder().setLongSerializationPolicy(LongSerializationPolicy.STRING)
-      .registerTypeAdapter(LongAdder.class, new TypeAdapter<LongAdder>() {
+      .disableHtmlEscaping().registerTypeAdapter(LongAdder.class, new TypeAdapter<LongAdder>() {
```

Comment:
Can we have separate method `createGsonWithDisableHtmlEscape()` for `RpcServer.logResponse()` since this one is being used at so many places?

Reply:
good idea